automation/check-patch.sh

```diff
@@ -115,16 +115,15 @@ run() {
     local provider="${PROVIDER:-lago}"
     local args=("prefix=$run_path")
     local inventory_file="$(realpath inventory)"
-    local storage_role="${STORAGE_ROLE:-storage-none}"
+    local storage_role="${STORAGE_ROLE:-storage-glusterfs}"
```

Comment:
Do not make this change. We are still waiting for #189 to be resolved so that we are working on any fixes in one place.

Reply:
I don't see a description of an issue in https://github.com/kubevirt/kubevirt-ansible/pull/189
Can you please explain what's blocking us?

Reply:
When I first made the comment the CI was still not passing. :) I thought it was going to be a much tougher battle, so I'm glad this resolves it!